server/app/graphql/types/gene_attribute_type.rb

```diff
@@ -0,0 +1,8 @@
+module Types
+    class GeneAttributeType < Types::BaseObject
+        field :id, ID, null: false
+        field :gene_id, String, null: false
```

Comment:
We should maybe represent the foreign key fields as being of type ID instead of String just to be consistent